[FIX] Update code for new rq version, 1.0

destanyol opened the pull request:
In the new version of rq, the function `get_failed_queue` has been deprecated.
This is a **non-tested update** of the code following the rq changelog of the v1.0 version.

https://github.com/rq/rq/blob/master/CHANGES.md

https://github.com/rq/rq/pull/1009/files?file-filters%5B%5D=.md&file-filters%5B%5D=.py#diff-97aa2f84df68bff74ea8272c7a176f99

destanyol committed:
FIX: get_failed_queue -> FailedJobRegistry

Commit:
Merge branch 'api_v5' into fix_redis_rq_v1

jaumef commented on oorq/bin/requeue_failed.py:
```diff
@@ -11,10 +12,10 @@
 pq = Queue(name=PERMANENT_FAILED)
 
-for job in fq.jobs:
+for job in pq.failed_job_registry:
```
```suggestion
for job in fq.failed_job_registry:
```

guilleJB replied:
`pq` is permantent queue not failed queue. Use `fq`

Commit:
Update oorq/bin/requeue_failed.py

Fix typo changing the used queue from fp to pq (failed → permanent)
So it's failed again

Commit:
Merge branch 'api_v5' into fix_redis_rq_v1

jaumef commented:
This PR must be on hold due to other requirements:

- rq-dashboard current version isn't compatible: `rq-dashboard 0.4.0 has requirement rq<1.0.0,>=0.3.8`
- redis<3.0 isn't compatible: `rq 1.0 has requirement redis>=3.0.0`

ecarreras commented:
I'm going to closes this because all is implemented in #83 Thanks for you work @destanyol
